strawberryfields/ops.py

```diff
@@ -2285,6 +2285,43 @@ def __str__(self):
         return temp
 
 
+class Ggate(Gate):
+    r"""General N-mode Gaussian gate.
+
+    The general N-mode Gaussian gate can be parametrized by a symplectic matrix S and a displacement parameter d.
+
+    .. math::
+        G(S,d)
+
+    Args:
+
+        S (array): symplectic matrix
+        d (array): vector of displacement parameters
+
+    .. details::
+
+        .. admonition:: Definition
+            :class: defn
+
+        .. math::
+            & cov = S*cov*S^T,\\
+            & mean = mean + d.
+
+    """
+    ns = None
```

Comment:
Why is this line needed? Seems like `self.ns` is set in the `__init__` method. Is `Ggate.ns` at any point called (would only return `None` either way). 🤔

Reply:
safe to remove? (I've seen `.ns` exists elsewhere)